storage/innobase/xtrabackup/doc/source/innobackupex/parallel_copy_ibk.rst

```diff
@@ -1,39 +0,0 @@
-.. _parallel-ibk:
-
-=================================
- Accelerating the backup process 
```

Comment:
This section remains valid for xtrabackup. Option names for --rsync and --parallel and --compress-threads are the same.

Reply:
Moreover, there is a --parallel option for --copy-back in xtrabackup which can be mentioned here as well.